daemon/emer-machine-id-provider.c

```diff
@@ -333,35 +325,29 @@ read_machine_id (EmerMachineIdProvider *self)
   g_autoptr(GError) local_error = NULL;
   uuid_t id;
 
-  if (!read_one_machine_id (priv->override_path, id, &local_error))
+  if (!read_one_machine_id (priv->tracking_id_path, id, &local_error))
     {
       if (g_error_matches (local_error, G_IO_ERROR, G_IO_ERROR_NOT_FOUND))
         {
-          g_debug ("Override machine id file %s does not exist, trying default.",
-                   priv->override_path);
+          g_debug ("Override machine id file %s does not exist.",
+                   priv->tracking_id_path);
```

Comment:
Perhaps we should call `write_tracking_id_file` here, rather than in constructed?

Reply:
Good idea.

However, after this change, to detect that the tracking-id-file initially doesn't exist, I had to change 
`G_IO_ERROR_NOT_FOUND` to `G_FILE_ERROR_NO_ENT` in `read_machine_id()`  . I debugged this when the test supporting this change was failiing for no obvious reasons.

I will update the PR tomorrow.